packages/test/test-utils/src/localLoader.ts

```diff
@@ -50,10 +51,14 @@ export function createLoader(
 ): IHostLoader {
     const codeLoader: ICodeLoader = new LocalCodeLoader(packageEntries);
 
+    const driver = typeof getFluidTestDriver === "function" ? getFluidTestDriver() : undefined;
+    const logger = typeof getTestLogger === "function" ? getTestLogger() : undefined;
```

Comment:
Hmm should we be consistent between using `?.()` (hoping it's either a function or undefined), v. here where you're more paranoid? I feel like it's fine to crash if someone sets `(global as any).getTestLogger = 5`...

Reply:
Actually I kind of like just assuming it's going to be there and return the right thing and get rid of all the `?` checks of any kind (like you do in most of the files above)

Reply:
it breaks some tests in table that use the test provider, but are not proper e2e tests